examples/step-37/step-37.cc

```diff
@@ -841,11 +841,13 @@ namespace Step37
         MatrixFree<dim,double>::AdditionalData::none;
       additional_data.mapping_update_flags = (update_gradients | update_JxW_values |
                                               update_quadrature_points);
+      std_cxx11::shared_ptr<MatrixFree<dim,double> >
+        system_mf_storage(new MatrixFree<dim,double>());
```

Comment:
No, it should be the same (at least for a big object), but `make_shared` involves 11 additional characters.